src/backend/src/utils/reimbursement-requests.utils.ts

```diff
@@ -15,12 +15,37 @@ export interface ReimbursementProductCreateArgs {
   wbsElementId: number;
 }
 
+export interface ReimbursementReceiptCreateArgs {
+  name: string;
+  googleFileId: string;
+}
+
 export interface ReimbursementProductCreateArgs {
   name: string;
   cost: number;
   wbsElementId: number;
 }
 
+/**
+ * This function removes any deleted receipts and adds any new receipts
+ * @param receipts the new list of receipts to compare against the old ones
+ * @param currentReceipts the current list of receipts on the request that's being edited
+ */
+export const updateReceiptPictures = async (receipts: ReimbursementReceiptCreateArgs[], currentReceipts: Receipt[]) => {
```

Comment:
if this only removes receipts then i think it should be named to reflect that

also `newReceipts` instead of just `receipts`